libs/environment/src/hooks/use-environment.tsx

```diff
@@ -42,14 +102,15 @@ export const EnvironmentProvider = ({
     <EnvironmentContext.Provider
       value={{
         ...environment,
-        configStatus,
+        networkError,
         setNodeSwitcherOpen: () => setNodeSwitcherOpen(true),
       }}
     >
-      {config && (
+      {config && isNodeSwitcherOpen && (
         <NodeSwitcherDialog
-          dialogOpen={isNodeSwitcherOpen}
-          toggleDialogOpen={setNodeSwitcherOpen}
+          dialogOpen={true}
```

Comment:
question: why do we now have to set this to true?

Reply:
true, this was from some leftover refactor. moved things around a bit to make this better. thanks!